### ISSUE TITLE
VictoriaLogs::cannot add rows into storage in read-only mode

### ISSUE THREAD

**Issue by tripitakav** [VictoriaMetrics/VictoriaMetrics#5112]
### Describe the bug

![image](https://github.com/VictoriaMetrics/VictoriaMetrics/assets/73687556/13bd98f5-4791-4c57-b5b7-d01f7b651d4a)

![image](https://github.com/VictoriaMetrics/VictoriaMetrics/assets/73687556/4bd2bc5c-33fd-4df8-b9c1-b1b98e573f44)

### To Reproduce

```yaml
apiVersion: apps/v1
kind: StatefulSet
metadata:
  labels:
    app: vmlog
  name: vmlog
  namespace: ops
spec:
  serviceName: vmlog
  selector:
    matchLabels:
      app: vmlog
  replicas: 1
  template:
    metadata:
      labels:
        app: vmlog
    spec:
      containers:
        - name: vmlog
          image: "docker.io/victoriametrics/victoria-logs:v0.4.0-victorialogs"
          imagePullPolicy: "IfNotPresent"
          args:
            - "--storageDataPath=/storage"
            - --retentionPeriod=4w
          ports:
            - name: http
              containerPort: 9428
          resources:
            limits:
              cpu: "2"
              memory: 10000Mi
            requests:
              cpu: 500m
              memory: 3000Mi
          volumeMounts:
            - name: vmlog-volume
              mountPath: /storage
      terminationGracePeriodSeconds: 60
  updateStrategy:
    type: RollingUpdate
  volumeClaimTemplates:
    - metadata:
        labels:
          app: vmlog-volume
        name: vmlog-volume
      spec:
        accessModes:
          - ReadWriteOnce
        resources:
          requests:
            storage: 500Gi
---
apiVersion: v1
kind: Service
metadata:
  labels:
    app: vmlog
  name: vmlog
  namespace: ops
spec:
  clusterIP: None
  ports:
    - port: 9428
      targetPort: http
      protocol: TCP
      name: http
  selector:
    app: vmlog
```

### Version

docker.io/victoriametrics/victoria-logs:v0.4.0-victorialogs

**Comment by zekker6** [comment]
Hello @tripitakav 
VictoriaLogs release [v0.4.0](https://docs.victoriametrics.com/VictoriaLogs/CHANGELOG.html#v040) added support of switching storage to read-only mode when free disk space available is lower than value specified by `-storage.minFreeDiskSpaceBytes` (by default it is `10Mb`).

Your metrics suggest that there is 99.9% space available from 500Gb volume and VictoriaLogs still switched to read-only which is not expected.
Could I ask you to exec into the pod and check filesystem availability there? (by using `df`/`lsblk` or something similar)

**Comment by zekker6** [comment]
@tripitakav This should be fixed in https://github.com/VictoriaMetrics/VictoriaMetrics/pull/5113, in the meanwhile could you try running `docker.io/victoriametrics/victoria-logs:heads-vlogs-fix-free-space-check-0-g650d72f33` and check if that helps?

**Comment by tripitakav** [comment]
@zekker6 I have no problem using `docker.io/victoriametrics/victoria-logs:heads-vlogs-fix-free-space-check-0-g650d72f33` Thanks！

**Comment by valyala** [comment]
The issue has been fixed in [v0.4.1-victorialogs](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v0.4.1-victorialogs). Closing the bug then.
